dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf

```diff
@@ -85,3 +85,15 @@ module "uswitch-reporter" {
   produce_topics   = [kafka_topic.uswitch_events_v1.name]
   cert_common_name = "customer-proposition/uswitch-reporter"
 }
+
+module "es-indexer-uswitch-data-v1" {
```

Comment:
you need to specify the consumer-group as well, like we did here https://github.com/utilitywarehouse/kafka-cluster-config/blob/bce437d40fad07523bb6741fea066f4a74129150/dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf#L52

Reply:
ah ok. So in this case would it be `es-indexer-uswitch.events.v1`? or does in need the namespace prefix as well?

Reply:
i think we should/are supposed to prefix consumer groups with the namespace, so `customer-proposition.es-indexer-uswitch.data.v1` (and the equivalent for `uswitch.events.v1`)